dashboard/app/views/layouts/_header.html.haml

```diff
@@ -12,6 +12,13 @@
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
 .header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if show_bug_links)}
+  - if current_user && script_level
+    #progresswarning
+      %a{href: "http://teacherblog.code.org/", target: "_blank"}
```

Comment:
How do you mean?  It was suggested because that's where we're putting a post about this, and we can also do newer posts whenever we like, if things change.